python/pyarrow/_parquet.pyx

```diff
@@ -481,14 +524,17 @@ cdef class RowGroupMetaData(_Weakrefable):
 
```

Comment:
There are some other methods above that aren't documented, should we document those?